dpdata/system.py

```diff
@@ -631,7 +632,44 @@ def from_gromacs_gro(self, file_name) :
         file_name : str
             The input file name
         """
-        self.data = dpdata.gromacs.gro.file_to_system_data(file_name)
+        self.data = dpdata.gromacs.gro.file_to_system_data(file_name, format_atom_name=format_atom_name)
+
+    @register_to_funcs.register_funcs("gromacs/string")
```

Comment:
The format `gromacs/string` is misleading. I would recommend to merge it into`gromacs/gro`. The interface of `gromacs/gro` can be revised to accept `file_name=None` in case one does not what to write a file, and returns the file content as a string.